flink-doris-connector/pom.xml

```diff
@@ -245,7 +245,7 @@ under the License.
         <dependency>
             <groupId>com.ververica</groupId>
             <artifactId>flink-sql-connector-mysql-cdc</artifactId>
-            <version>2.3.0</version>
+            <version>2.4.1</version>
```

Comment:
Will 2.4.1 conflict with oracle2.3.0？

Reply:
This problem occurs when running the `CdcOraclelSyncDatabaseCase` class alone